### PR TITLE
added description for Id parameter

### PR DESCRIPTION
**Pull request by Swathi-Birlangi**

### CODE REVIEW

File: api-reference/beta/api/calendar-list-events.md [hunk 1]
```diff
@@ -118,7 +126,162 @@ Content-length: 354
...
+        "time": "datetime-value"
+      },
+      "iCalUId": "iCalUId-value",
+      "reminderMinutesBeforeStart": 99,
+      "isReminderOn": true
+    }
+  ]
+}
+```
+This method supports some of the OData query parameters to help customize the response. For example, to filter the access packages with subject, include a filter such as `$filter=startsWith(subject,'All')` in the query. Simultaneously, filter to get all emails from a specific address received by the signed-in user `messages?$filter=from/emailAddress/address eq 'admin@contoso.com`.
```
Comment (angelgolfer-ms) [inline api-reference/beta/api/calendar-list-events.md:161]: Line 161: Delete this paragraph please.

File: api-reference/beta/api/calendar-list-events.md [hunk 2]
```diff
@@ -118,7 +126,162 @@ Content-length: 354
...
+
+### Example 2: Filter with subject and emails from a specific address
```
Comment (angelgolfer-ms) [inline api-reference/beta/api/calendar-list-events.md:163]: Line 163: 
Retitle this to say "Example 2: Get events by filtering on the subject property"

Line 165: Delete this line!

Add the sentence "This example gets events in the user's primary calendar by filtering on their subject starting with "All".

Line 167 and 178: Change them to level 4 heading, not level 5.

File: api-reference/v1.0/api/calendar-list-events.md [hunk 3]
```diff
@@ -116,6 +123,129 @@ Content-length: 354
 }
 ```
 
+### Example 2: Filter with subject and emails from a specific address
```
Comment (angelgolfer-ms) [inline api-reference/v1.0/api/calendar-list-events.md:126]: Example 2: please apply the comments I gave for the beta topic.

**Review by angelgolfer-ms — CHANGES_REQUESTED** [review]
As I mentioned in our Teams chat (9/23 PST), I suggest the following for this PR:
- Keep this PR 13397 addressing Github issue 12400.
- For Github issue 9685, I don't think adding an example is what the customer is looking for. On the other hand, just to enrich the topic, we can add an example that shows this query URL:

GET https://graph.microsoft.com/beta/me/calendar/events?$filter=startsWith(subject,'All')

Swathi can please incorporate the comments I added to the beta calendar-list-events.md topic. Do the similar to the v1 topic.

Do a separate docs PR to address Github issue 9685, as I suggested in our Teams chat.

File: api-reference/beta/api/calendar-list-events.md (hunk 1 above)
Comment (Swathi-Birlangi) [reply api-reference/beta/api/calendar-list-events.md:161]: Deleted this content in beta and v1.0

File: api-reference/beta/api/calendar-list-events.md (hunk 2 above)
Comment (Swathi-Birlangi) [reply api-reference/beta/api/calendar-list-events.md:163]: Line 163:  Renamed in both beta and v1.0

Line 165: deleted in both beta and v1.0

Added the sentence in both beta and v1.0

Line 167 and 178: changed the heading to level 4 in both beta and v1.0

File: api-reference/v1.0/api/calendar-list-events.md (hunk 3 above)
Comment (Swathi-Birlangi) [reply api-reference/v1.0/api/calendar-list-events.md:126]: Applied in beta and v1.0

**Pull request merged**